doc/examples/config/config-storage/README.md

```diff
@@ -71,6 +71,12 @@ The data corresponding to the path contains the following fields:
 * `value`: any string.
 * `mod_revision`: unsigned number, last `revision` at which this
 value modification occurred.
+* `deadline`: unsigned number, time until expiration. This field is present
+  since Tarantool version 3.2.
```

Comment:
Sorry for nitpicking, but all the other fields (`path`, `value` and `mod_revision`) are present in the `config.storage.get` response. It may give a false assumption that the deadline is present in this response are well.

Since the deadline doesn't shown in any response, I wouldn't mention it here, at least not in this form.

We can say that a TTL may be associated with the data, but we shouldn't give any certain name. For example:

> * `path`: <...>
> * `value`: <...>
> * `mod_revision`: <...>
> * An optional time to live, after which the data is considered as deleted (since Tarantool 3.2).

Reply:
Updated.

Reply:
Thanks!